pages/dkp/kaptain/1.2.0/install/konvoy/index.md

```diff
@@ -61,7 +61,7 @@ For cloud installations, scaling out can be limited by resource quotas.
         - name: knative
           enabled: true
     ```
-
+<!-- Do we need to update anything on the file location above? update/decompress the link?-->
```

Comment:
No this is a repository that can't change for older versions - so the URL will stay the same. This particular reference is only for the knative install - no MinIO included here. The installs from here happen automatically. This is the older version of what is now catalog and Kommander applications.